target/linux/ramips/dts/mt7620a_domywifi_dw22d.dtsi

```diff
@@ -0,0 +1,151 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "domywifi,dw22d", "ralink,mt7620a-soc";
+	model = "DomyWifi DM202/DM203/DW22D";
+
+	aliases {
+		led-boot = &led_sys_red;
+		led-failsafe = &led_sys_red;
+		led-running = &led_sys_green;
+		led-upgrade = &led_sys_red;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio0 13 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		internet_green {
+			label = "green:internet";
+			gpios = <&gpio1 0 GPIO_ACTIVE_HIGH>;
+		};
+
+		internet_red {
+			label = "red:internet";
+			gpios = <&gpio1 5 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_sys_green: sys_green {
```

Comment:
Why not use "system" as well?

Reply:
This name seems too long, I'll change it.

Reply:
?

Reply:
If I understand you correctly, what you mean is to replace "sys_green " with "system"? The system LED has two colors so I add additional suffix.

Reply:
No, I mean using system_green and green:system instead of the abbreviated version. Is there any reason for the abbreviation?

Reply:
Ok, fixed.